src/com/trollworks/gcs/widgets/outline/MultiCell.java

```diff
@@ -205,10 +205,22 @@ public Cursor getCursor(MouseEvent event, Rectangle bounds, Row row, Column colu
         return Cursor.getDefaultCursor();
     }
 
+    /** Allow a satified row to display a tooltip */
     @Override
     public String getToolTipText(Outline outline, MouseEvent event, Rectangle bounds, Row row, Column column) {
         ListRow theRow = (ListRow) row;
-        return theRow.isSatisfied() ? null : theRow.getReasonForUnsatisfied();
```

Comment:
Any reason you didn't just replace `null` with `getToolTip(row, column)` on this line?

Reply:
None.